packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -51,6 +56,34 @@ export async function activate(context: ExtensionContext): Promise<void> {
     clientOptions
   );
 
+  const jsonSchemaConfig:
+    | WorkspaceConfiguration
+    | undefined = workspace.getConfiguration().get("json.schemas");
+  if (
+    process.env["THEIA_PARENT_PID"] !== undefined &&
+    isArray(jsonSchemaConfig)
+  ) {
+    const providedManifestSchemaConfigs = find(jsonSchemaConfig, (_) => {
```

Comment:
`providedManifestSchemaConfigs` can be renamed to `hasManifestJsonConfig` (boolean) if you
also inspect the result of the find `!== undefined` in this same statement.

We don't care about the result, only its existence or lack thereof.